nservicebus/licensing/index.md

```diff
@@ -35,6 +35,4 @@ If your version is not listed or has expired but you would like support, please
 
 ### NServiceBus 3.x
 
-|  Version  | Released        | Support Expires |
-|:---------:|-----------------|:---------------:|
-|   3.3.0   | 2012-10-29      |   2015-10-29    |
\ No newline at end of file
+Official support for version 3 has expired as of October 10 2015, please [contact us](http://particular.net/contactus) if you are interested in extended support or upgrade help.
```

Comment:
How about:
We no longer support version 3 as of October 10, 2015. Please [contact us](http://particular.net/contactus) if you require assistance.